modules/cms/lang/zh-tw/lang.php

```diff
@@ -168,8 +171,8 @@
         'content' => '內容',
         'hidden' => '隱藏',
         'hidden_comment' => '隱藏頁面只能被登錄的後台使用者訪問.',
-        'enter_fullscreen' => '進入全屏模式',
-        'exit_fullscreen' => '退出全屏模式'
+        'enter_fullscreen' => '進入全螢幕模式',
+        'exit_fullscreen' => '退出全螢幕模式'
```

Comment:
Please make sure trailing commas are always used in language files, this makes it easier for future enhancements to be made